src/platform/assets/components/MediaImageBottom.vue

```diff
@@ -4,18 +4,20 @@
       class="m-0 line-clamp-1 text-sm font-bold text-zinc-900 dark-theme:text-white"
       :title="asset.name"
     >
-      {{ fileName }}
+      {{ truncateFilename(fileName) }}
```

Comment:
It feels like there's a lot of duplication across these, could they be joined into a single component?